html/rules.html

```diff
@@ -1,75 +1,75 @@
+<p>• Aggressor is a raider (Motivation: Raiding and stealing, etc) and tells the victim to get on the ground. Victim replies “Please don’t hurt me” and lays on the ground. The raider now has full control over the victim and may do whatever they please with them, however killing them on the spot without further roleplay will warrant a bwoink.</p>
+<p>• Aggressor is NCR (Motivation: To deal with a raider gang.) and tells them to surrender or die. Some of the Raiders Surrender, but the rest are gunned down. The NCR then detains the raiders that surrendered.</p>
 <p>
-    <br /><strong>C.1. NCR and Legion Conflict</strong></p>
-<p>Given that approach rules are followed combat is allowed at any point;
-    <br />And is expected to be initiated, the NCR and Legion are at war.</p>
-<p>To engage with your enemy at every point isn't required but it is entirely within reason.</p>
-<p>Retreating is fine if you can&rsquo;t win, but remember, actively avoiding escalation looks bad from the admins perspective.</p>
-<p>Escalation rules apply.</p>
+    <br /><strong>General Conflict</strong></p>
+<p>1. Settlers and Wastelanders should not be starting fights and trying to rob people.</p>
+<p>2. Settlers and Wastelanders may react to theft and other petty crimes appropriately, which in the Wasteland means shooting them.</p>
+<p>3. Raiders may escalate for motivations such as theft, or being tasked with killing someone.</p>
+<p>4. Mercenaries may escalate for the purposes of killing targets, or defending someone.</p>
+<p>Eg: Your house is being broken into, and you shoot the person breaking in. Someone steals a locker from you as you travel.</p>
+<p>Eg: If someone is running from you and not letting you escalate, you can fire upon them as this is fleeing conflict. Be mindful as they may not have noticed, so point at them aggressively.</p>
 <p>
-    <br /><strong>D. Alignment and Siding</strong></p>
-<p>1.Wastelanders and raiders are allowed to join and side with any of the factions.</p>
-<p>2. No hostile action is allowed to be taken while siding with a faction unless you are in their full uniform and attire. Unless you express your verbal allegiance during escalation, you will not be considered valid.</p>
-<p>3. When you have joined a faction, you should wear as much of their uniform as is available, or verbally side with them, when situations are escalating.</p>
-<p>4. All NCR/BOS/Legion Roles, are beholden to IC Rules, and in the case of their Leaders/Higher Ranks, OOC Rules.</p>
-<p>5. Repeatedly breaking IC rules can result in admin intervention.</p>
+    <br /><strong>Uniform and Siding</strong></p>
+<p>1. Faction uniforms are to be kept at all times unless deserting or infiltrating. (Expect consequences if you desert your respective faction!)</p>
+<p>2. To participate in a faction raid/conflict, you MUST be in that factions uniform, or be clearly identifiable through the use of the Faction Tabard item.</p>

```

Comment:
This item does not exist. Refer to the armor and helmet requirement further up.

Reply:
It will exist soon

Reply:
Then add this line when it does exist. Jumping the gun is why we had rules referring to bitter drink.